native_client/definitions.mk

```diff
@@ -63,9 +63,19 @@ PYTHON_PACKAGES := numpy${NUMPY_BUILD_VERSION}
 endif
 
 ifeq ($(TARGET),rpi3)
+PYVER := $(shell python -c "import platform; maj, min, _ = platform.python_version_tuple(); print(maj+'.'+min);")
+ifeq ($(PYVER), 3.7)
+RASPBIAN    ?= $(abspath $(NC_DIR)/../multistrap-raspbian-buster)
+NUMPY_INCLUDE        := NUMPY_INCLUDE=$(RASPBIAN)/usr/include/python$(PYVER)m/
```

Comment:
this might not always be true, I remember that numpy includes were having a bit of fun and being inside a different path than the python version (hence the hardcoded path), is that fixed?

Reply:
Yep it can either be in `/usr/include/numpy` or in `/usr/include/python3.7m/numpy`.

In the target chroot, we can use `numpy.get_include()` from python to get the include path that numpy needs but I’m not sure it’s wise to make it dynamic here… can we even import the target’s numpy from the CI? 🤔

Reply:
@lissyx / @wasertech should i keep the hardcoded version tehn

Reply:
Unless someone as a better idea, yes because it works